lib/modules/datasource/metadata.ts

```diff
@@ -4,7 +4,8 @@ import parse from 'github-url-from-git';
 import { DateTime } from 'luxon';
 import * as hostRules from '../../util/host-rules';
 import { regEx } from '../../util/regex';
-import { parseUrl, validateUrl } from '../../util/url';
+import { parseUrl, trimTrailingSlash, validateUrl } from '../../util/url';
+import { detectPlatform } from '../../util/common';
```

Comment:
probably needs import reorder

Reply:
its weird, the lint Passed, then i  Updated branch from main, suddenly i get this , ill run lint fix again i guess.

Reply:
someone changed the classes, there's a conflict im fixing